applications/potree-converter/tapisjob_app.sh

```diff
@@ -0,0 +1,13 @@
+set -x
+# Run the script with the runtime values passed in from the job request
+
+echo "Potree Converter 2.1.1"
+
+/opt/PotreeConverter/build/PotreeConverter ${converterInput} -o ${_tapisExecSystemOutputDir} -m ${samplingMethod}
```

Comment:
would this need `--generate-page`?

> -p [ --generate-page ]  Generate a ready to use web page with the given name

Reply:
![Screenshot 2024-05-17 at 10 06 25 AM](https://github.com/TACC/WMA-Tapis-Templates/assets/8287580/301c92e3-3c66-44e3-8626-744231ee54e8)
 
Previously, there was an extra arguments input field?  should that be done instead of the `samplingMethod` (unless samplingMethod was a requirement from somebody).  and if we do that we can improve things by always adding `--generate-page index` and then add any extra.  then we could avoid having all the UI text about "To use the PotreeViewer App, you must add -p index in the arguments".

Reply:
Yes, completely agreed, had to dig a bit more into their code to find the command options - I will update with a reference to their commandline args. Also agreed on keeping the `generate-page` arg in there by default.